mozartdata/transforms/fact/inbound_shipment_item.sql

```diff
@@ -1,14 +1,23 @@
 SELECT
```

Comment:
Not major but just adding some comments to the top.

```suggestion
/*
     Purpose: To link the inbound shipment items with our dim.product table so we can show information 
                         related to products in that table easily in future reporting.
     Primary Key: inbound_shipment_item_id_ns
*/

SELECT
```